R/utils.R

```diff
@@ -165,3 +169,106 @@ random_name <- function(prefix = "") {
   name <- paste0(sample(vals, 10, replace = TRUE), collapse = "")
   paste0(prefix, "odbc_", name)
 }
+
+# apple + spark drive config (#651) --------------------------------------------
+configure_spark <- function(call = caller_env()) {
+  if (!is_macos()) {
+    return(invisible())
+  }
+
+  unixodbc_install <- locate_install_unixodbc()
+  if (identical(unixodbc_install, character(0))) {
```

Comment:
```suggestion
  if (length(unixodbc_install) == 0) {
```

I think that makes the interface a bit clearer to me.